### ISSUE TITLE
SBGN-ML comes back empty

### ISSUE THREAD

**Issue by jvwong** [PathwayCommons/sbgnml-to-cytoscape#21]
I tried to convert the following [SBGN-ML for a complex]( http://www.pathwaycommons.org/pc2/get?format=sbgn&uri=http://pathwaycommons.org/pc2/Complex_4839c52f0b319f9132e85d41891024c4):

```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<sbgn
	xmlns="http://sbgn.org/libsbgn/0.2">
	<map language="process description">
		<glyph class="complex" id="Complex_4839c52f0b319f9132e85d41891024c4">
			<label text="TP53 protein/GDF15 promoter complex"/>
			<bbox w="54.0" h="34.0" x="-2.0" y="-2.0"/>
			<glyph class="macromolecule" id="Protein_b037e4d948a09a80bd497c39d035e1eb_Complex_4839c52f0b319f9132e85d41891024c4">
				<label text="TP53"/>
				<bbox w="48.0" h="25.0" x="0.0" y="0.0"/>
			</glyph>
			<glyph class="nucleic acid feature" id="DnaRegion_0e23425e554065410c00a59c6bf7a878_Complex_4839c52f0b319f9132e85d41891024c4">
				<label text="GDF15"/>
				<bbox w="50.0" h="30.0" x="0.0" y="0.0"/>
				<glyph class="unit of information">
					<label text="ct:DNA"/>
					<bbox w="35.0" h="10.0" x="0.0" y="0.0"/>
				</glyph>
			</glyph>
		</glyph>
	</map>
</sbgn>
```

but the result is empty: 

```js
{
  "nodes": [],
  "edges": []
}
```

I noticed this from the app-ui pathway viewer originally.

**Comment by d2fong** [comment]
@jvwong I published a new release (4.0.2).  Can you try it?

**Comment by jvwong** [comment]
Yup looks OK to me for this example:

```js
{
  "nodes": [
    {
      "data": {
        "id": "Complex_4839c52f0b319f9132e85d41891024c4",
        "class": "complex",
        "label": "TP53 protein/GDF15 promoter complex",
        "parent": "",
        "clonemarker": false,
        "stateVariables": [],
        "unitsOfInformation": [],
        "bbox": {
          "x": 25,
          "y": 15,
          "w": 54,
          "h": 34
        }
      }
    },
    {
      "data": {
        "id": "DnaRegion_0e23425e554065410c00a59c6bf7a878_Complex_4839c52f0b319f9132e85d41891024c4",
        "class": "nucleic acid feature",
        "label": "GDF15",
        "parent": "Complex_4839c52f0b319f9132e85d41891024c4",
        "clonemarker": false,
        "stateVariables": [],
        "unitsOfInformation": [
          {
            "id": null,
            "class": "unit of information",
            "label": {
              "text": "ct:DNA"
            }
          }
        ],
        "bbox": {
          "x": 25,
          "y": 15,
          "w": 50,
          "h": 30
        }
      }
    },
    {
      "data": {
        "id": "Protein_b037e4d948a09a80bd497c39d035e1eb_Complex_4839c52f0b319f9132e85d41891024c4",
        "class": "macromolecule",
        "label": "TP53",
        "parent": "Complex_4839c52f0b319f9132e85d41891024c4",
        "clonemarker": false,
        "stateVariables": [],
        "unitsOfInformation": [],
        "bbox": {
          "x": 24,
          "y": 12.5,
          "w": 48,
          "h": 25
        }
      }
    }
  ],
  "edges": []
}
```
